<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dossier</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nom du dossier</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="namedossier">
     <property name="inputMask">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Le nom du dossier</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Type de dossier</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="typeDossier">
     <item>
      <property name="text">
       <string>Appartement</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Terrain à batir(TAB)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Maison</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lotissement</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Garage</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>VEFA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SCI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Parcelle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Terrain</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Provenance</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="provanceDrop">
     <item>
      <property name="text">
       <string>Agence</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Autre</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="statusAvantContrat">
     <property name="text">
      <string>Statut Avant Contrat</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Notaire Vendeur</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="NotaireVendeur"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Notaire Acquéreur</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="8" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Besoin de Prêt</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Oui</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Non</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Je ne sais pas</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Commentaires</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QTextEdit" name="textComment"/>
   </item>
   <item row="11" column="0">
    <widget class="QDialogButtonBox" name="validatenewdossier">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="statusAvantContratComb">
     <item>
      <property name="text">
       <string>En cours - Agence</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OK - Agence</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>validatenewdossier</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>validatenewdossier</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
